File: JavaScript/PrimeX.js
const NL = 100000000;

console.log("========\n PrimeX\n========\n");

const fs = require("fs");
const writeStream = fs.createWriteStream("primes.txt");

const N = NL + 2;
let primes = Array(N).fill(true);

for (let i = 2; i < 3; ++i) {
    if (primes[i]) {
        writeStream.write(i + '\n');

        for (let j = i; j < N; j += i) {
            primes[j] = false;
        }
    }
}

for (let i = 3; i < N; i += 2) {
    if (primes[i]) {
        writeStream.write(i + '\n');

        for (let j = i; j < N; j += i) {
            primes[j] = false;
        }
    }
}

primes = null;
writeStream.close();

console.log("Done :D\nCheck your folder for the results\n");

File: Julia/PrimeX.jl
NL = 1000000000

print("========\n PrimeX\n========\n\n")

file = open("primes.txt", "w")

N = NL + 2
primes = fill(true, N)

for i = 2:1:2
    if primes[i]
        write(file, string(i) * "\n")

        for j = i:i:N
            primes[j] = false;
        end
    end
end

for i = 3:2:N
    if primes[i]
        write(file, string(i) * "\n")

        for j = i:i:N
            primes[j] = false;
        end
    end
end

close(file)

print("Done :D\nCheck your folder for the results\n\n")